src/storage/storage.go

```diff
@@ -12,6 +13,13 @@ import (
 
 const currentVersion = 4
```

Comment:
Should we up the database version?

Reply:
Yes, good catch. Otherwise the new tables will be missing.